otbenchmark/mini_Benchmark.py

```diff
@@ -0,0 +1,182 @@
+        inputVector = myEvent.getAntecedent()
+        myDistribution = inputVector.getDistribution()
+        g = Problem.getEvent().getFunction()
+        algoSORM = ot.SORM(myCobyla, myEvent, myDistribution.getMean())
+        initialNumberOfCall = g.getEvaluationCallsNumber()
+        algoSORM.run()
+        resultSORM = algoSORM.getResult()
+        numberOfFunctionEvaluationsSORM = (
+            g.getEvaluationCallsNumber() - initialNumberOfCall
+        )
+        return [resultSORM, numberOfFunctionEvaluationsSORM]
+
+    def MonteCarloSampling(Problem, NBS, cv):
+
+        myEvent = Problem.getEvent()
+        g = Problem.getEvent().getFunction()
+        experiment = ot.MonteCarloExperiment()
+        myMC = ot.ProbabilitySimulationAlgorithm(myEvent, experiment)
+        myMC.setMaximumOuterSampling(NBS)
+        myMC.setBlockSize(1)
+        myMC.setMaximumCoefficientOfVariation(cv)
+        initialNumberOfCall = g.getEvaluationCallsNumber()
+        myMC.run()
+        resultMC = myMC.getResult()
+        numberOfFunctionEvaluationsMonteCarlo = (
+            g.getEvaluationCallsNumber() - initialNumberOfCall
+        )
+        return [
+            resultMC,
+            numberOfFunctionEvaluationsMonteCarlo,
+            myMC.drawProbabilityConvergence(),
+        ]
+
+    def ImportanceSampling(Problem):
+        # Cobyla algorithm
+        myCobyla = ot.Cobyla()
+        # Resolution options:
+        eps = 1e-3
+        myCobyla.setMaximumEvaluationNumber(100)
+        myCobyla.setMaximumAbsoluteError(eps)
+        myCobyla.setMaximumRelativeError(eps)
+        myCobyla.setMaximumResidualError(eps)
+        myCobyla.setMaximumConstraintError(eps)
+
+        myEvent = Problem.getEvent()
+        inputVector = myEvent.getAntecedent()
+        myDistribution = inputVector.getDistribution()
+        g = Problem.getEvent().getFunction()
+        initialNumberOfCall = g.getEvaluationCallsNumber()
+        algoFORM = ot.FORM(myCobyla, myEvent, myDistribution.getMean())
+        algoFORM.run()
+        resultFORM = algoFORM.getResult()
+        standardSpaceDesignPoint = resultFORM.getStandardSpaceDesignPoint()
+        d = myDistribution.getDimension()
+        myImportance = ot.Normal(
+            standardSpaceDesignPoint, [1.0] * d, ot.CorrelationMatrix(d)
+        )
+
+        experiment = ot.ImportanceSamplingExperiment(myImportance)
+        algo = ot.ProbabilitySimulationAlgorithm(ot.StandardEvent(myEvent), experiment)
+        algo.setMaximumCoefficientOfVariation(0.1)
+        algo.setMaximumOuterSampling(50000)
+        algo.setConvergenceStrategy(ot.Full())
+        algo.run()
+        resultTirage = algo.getResult()
+        graph = algo.drawProbabilityConvergence()
+        numberOfFunctionEvaluationsTirage = (
+            g.getEvaluationCallsNumber() - initialNumberOfCall
+        )
+        return [resultTirage, numberOfFunctionEvaluationsTirage, graph]
+
+    def SubsetSampling(Problem):
```

Comment:
Please create new parameters for maximum outer sampling, coefficient of variation and block size.